index.html

```diff
@@ -6,8 +6,67 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7cbe454c49.js" crossorigin="anonymous"></script>
 </head>
 <body>
+    <header class="header-container">
+    <div class="sub-container">
+        <a href="#" class="logo-cont">Olamarx</a>
+      <button type="submit" class="header-font"><i class="fa fas-solid fa-bars font"></i> </button> 
```

Comment:
- [ ] Please make sure to stick to the original [figma]() design guidelines(fonts, margins, paddings, colors etc)
- [ ] Also, you should use the hamburger specified in the design. You can export it from figma and then include it in the project.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/158801323-ba587a21-4ae7-49d0-ae9f-91614f34b93e.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/158801414-7d81a81b-413c-4fff-b138-717f5f43bbf5.png)